src/canister/post_cache/src/data_model/mod.rs

```diff
@@ -22,6 +22,10 @@ pub struct CanisterData {
     pub posts_index_sorted_by_home_feed_score_v1: PostScoreHomeIndex,
     #[serde(default)]
     pub posts_index_sorted_by_hot_or_not_feed_score_v1: PostScoreHotOrNotIndex,
+    
+    #[serde(skip_deserializing, default)]
```

Comment:
Is skip_deserializing even required ? According to serde docs - 
`If the value is not present when deserializing, use the Default::default().`

Can we check locally by removing this and uncommenting post_upgrade and see if updates are working fine ? One deploy should suffice in that case

Steps:
1. Deploy `main` branch canisters
2. Run `./ic_repl_tests/all_tests.sh` . This will add data in canisters
3. Check post_cache get API
4. Update canisters -> upgrade_all_canisters.sh script
5. Check the Yral cursor feed API 

Can also be done in PocketIC .

Reply:
Wil test this. good idea!